C-Plus-Plus/graphs/DFS.cpp

```diff
@@ -0,0 +1,76 @@
+#include<bits/stdc++.h>
+using namespace std;
+//nodes 1 to n
+void createAdjList(vector<int> adj[], vector<vector<int>> &edges)
+{
+    int m = edges.size();
+    for(int i=0; i<m; i++)
+    {
+        adj[edges[i][0]].push_back(edges[i][1]);
+        adj[edges[i][1]].push_back(edges[i][0]);
+
+    }
+}
+
+void dfsVisit(vector<int> &visited, vector<int> adj[], int src)
+{
+    visited[src] = 1;
+    for(int i=0; i<adj[src].size(); i++)
+    {
+        if(!visited[adj[src][i]])
+        {
+            dfsVisit(visited, adj, adj[src][i]);
+        }
+    }
+    cout << src << " ";
+}
+
+void dfs(int n, vector<vector<int>> &edges)
+{
+    vector<int> adj[n+1];  
+    vector<int> visited(n+1, 0);
+    createAdjList(adj, edges);
+    for(int i=1; i<=n; i++)
+    {
+        if(!visited[i])
+            dfsVisit(visited, adj, i);
+    }
+
+}
+
+int main()
+{
+    int n, m;
+    cin >> n >> m;
```

Comment:
Use better variables and prompt user to enter values so that the program becomes more interactive.

Reply:
@atarax665 Done with the chages plz review